### PR TITLE
JS tests: Break out column helpers, helpers, and map functionality to modules

### LINKED ISSUE #2005: Add automated JS test for elections
Backfill test for Javascript we have developed - this is for the Election Pages.
1 pt for Andrew

-[ ] write test for elections

### PR DESCRIPTION
**Pull request by apburnes**
## Breaks out and tests modules and functions used to build elections page tables and maps

- Resolves #2005
- Moves functions into semantic module locations like `columns`, `helpers`, `maps`, and `tables`
- Add tests to those modules

## Impacted areas of the application
`static/js` modules and `elections.js` page

### CODE REVIEW

**Comment by johnnyporkchops** [comment]
I am just going to leave this open because it is not in the  07/16 release. That way @patphongs can be prompted to review it since she is listed as reviewer

**Comment by JonellaCulmer** [comment]
@patphongs This is one of the last holdouts of work from 6.3. Do you mind taking some time early this week to review and merge? See previous comments for more info. 

cc: @johnnyporkchops @apburnes

File: fec/fec/static/js/modules/helpers.js [hunk 1]
```diff
@@ -253,6 +253,26 @@ function buildUrl(path, query) {
     .toString();
 }
 
+function buildCycleUrl(candidate, cycle, path) {
+
+}
```
Comment (patphongs) [inline fec/fec/static/js/modules/helpers.js]: Looks like an empty function here. Is this an extra helper function that you meant to populate?

Comment (apburnes) [reply fec/fec/static/js/modules/helpers.js]: I think it's was something I was going to breakout but probably realized it was unnecessary. Good catch!! I'll clean it up and push an update.

**Comment by patphongs** [comment]
@apburnes There appears to be a conflict in this PR, could you please resolve it?

**Pull request merged**